canary/src/main/java/io/littlehorse/canary/metronome/MetronomeGetWfExecutor.java

```diff
@@ -0,0 +1,11 @@
+package io.littlehorse.canary.metronome;
+
+import io.littlehorse.canary.metronome.internal.LocalRepository;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class MetronomeGetWfExecutor {
+    public MetronomeGetWfExecutor(final LocalRepository repository) {
+        log.info("GetWf Metronome Started");
+    }
```

Comment:
Assuming that this is the "Work In Progress" part? And this will check everything in the RocksDB, and keep doing a `GetWfRun` until it is completed?

Reply:
yes, WIP